test/client/requestUtil.js

```diff
@@ -52,7 +52,25 @@ test('client RequestUtil', (t) => {
     })
     const serializer = requestUtil.serializer
 
-    t.plan(2)
+    t.plan(3)
+
+    t.test('#should list object', (t) => {
+      t.plan(3)
+
+      t.equals(true,
+        requestUtil.shouldListObject(0, proto.categories.BOOKMARKS),
+        `${t.name}: previous fetch time is empty - use SQS`)
```

Comment:
this uses listing from s3

Reply:
Thanks for finding this. 
I have made mistake in that string, but expected value `true` is still correct for this case.

Addressed  in  https://github.com/brave/sync/pull/363/commits/d19b9c5cbf77b33cfcc7f7d419805c9ad6ef54f1